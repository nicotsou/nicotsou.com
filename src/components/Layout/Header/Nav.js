import React from 'react'
import MenuIcon from '../../../images/menu-icon.svg'
import Icon from '../Icon'

const Nav = () => {
  return (
    <Icon>
      <MenuIcon />
    </Icon>
  )
}

export default Nav
