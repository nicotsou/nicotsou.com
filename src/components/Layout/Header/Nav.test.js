import React from 'react'
import { render, screen } from '@testing-library/react'
import Nav from './Nav'

describe('Nav', () => {
  it('renders without issues', () => {
    const { container } = render(<Nav />)
    expect(container).toMatchSnapshot()
  })
})
