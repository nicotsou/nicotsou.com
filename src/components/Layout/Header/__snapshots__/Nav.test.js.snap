// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders without issues 1`] = `
<div>
  <div
    class="sc-bdfBwQ dliorU"
  >
    <test-file-stub />
  </div>
</div>
`;
